src/static/css/nav-secondary.less

```diff
@@ -9,6 +9,10 @@
 @nav-secondary_link__disabled-outline:         @gray-20;
 @nav-secondary_link__disabled-border:          @nav-secondary_link__disabled-bg;
 
+// TODO: Refactor these variable to use existing CF font size variables.
+@font-size-small: 16px;
+@font-size-large: 18px;
```

Comment:
There are no existing CF font-size variables, other than `@base-font-size-px`. I'm not sure I see the value in creating new variables here.

It's outside the scope of this issue, but I think we need to better articulate our philosophy on units and values and how they're declared.